proto/kava/kavadist/v1beta1/params.proto

```diff
@@ -13,6 +15,36 @@ option (gogoproto.goproto_getters_all)  = false;
 message Params {
   bool            active  = 1 [(gogoproto.moretags) = "yaml:\"active\""];
   repeated Period periods = 3 [(gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"periods\""];
+  InfrastructureParams infrastructure_params = 4 [(gogoproto.nullable) = false];
+}
+
+message InfrastructureParams {
+  repeated Period infrastructure_periods = 1 [(gogoproto.castrepeated) = "Periods", (gogoproto.nullable) = false];
+  repeated CoreReward core_rewards = 2 [(gogoproto.castrepeated) = "CoreRewards", (gogoproto.nullable) = false];
+  repeated PartnerReward partner_rewards = 3 [(gogoproto.castrepeated) = "PartnerRewards", (gogoproto.nullable) = false];
+  option (gogoproto.goproto_stringer)        = true;
+}
+
+message CoreReward {
+  bytes   address      = 1                   [
+    (cosmos_proto.scalar) = "cosmos.AddressBytes",
+    (gogoproto.casttype)  = "github.com/cosmos/cosmos-sdk/types.AccAddress"
+  ];
+  string weight = 2 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  option (gogoproto.goproto_stringer)        = true;
+}
+
+message PartnerReward {
+  bytes   address      = 1                   [
+    (cosmos_proto.scalar) = "cosmos.AddressBytes",
+    (gogoproto.casttype)  = "github.com/cosmos/cosmos-sdk/types.AccAddress"
+  ];
+  cosmos.base.v1beta1.Coin rewards_per_second         = 2[(gogoproto.nullable) = false];
```

Comment:
should this be an int? Doesn't look like any other denoms are supported

Reply:
going to punt on this unless there's strong desire to change it (cc @rhuairahrighairidh )